esmvalcore/_task.py

```diff
@@ -367,10 +367,7 @@ def _write_ncl_settings(self):
             'run_dir',
             'plot_dir',
             'work_dir',
-            'output_file_type',
```

Comment:
Same here, don't delete these variables just yet and remove the block to add them back below.

Reply:
some change is needed here, because otherwise these settings will not be available as variable `diag_script_info@write_plots` etc, making it impossible to adjust diagnostic scripts to the new situtation before they are removed in 2.3. As an intermediate step, I thought I would make them available as both `diag_script_info@write_plots` and `config_user_info@write_plots`.